comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -24,37 +39,95 @@ public static bool IsDirectory(string path)
 		// Return the total size, in bytes, of all the files below the given directory
 		public static long GetTotalSize(string directory)
 		{
-			throw new NotImplementedException();
+            string[] files = Directory.GetFiles(directory, "*.*", SearchOption.AllDirectories);
+            long TotalSize = 0;
+            for (int i = 0; i < files.Length; i++)
+            {
+                TotalSize += GetFileSize(files[i]);
+            }
+            return TotalSize;
 		}
 
 		// Return the number of files (not counting directories) below the given directory
 		public static int CountFiles(string directory)
 		{
-			throw new NotImplementedException();
-		}
+            string[] files = Directory.GetFiles(directory, "*.*", SearchOption.AllDirectories);
+            int JustFiles = 0;
+            for (int i = 0; i < files.Length; i++)
+            {
+                if (IsDirectory(files[i]) == false)
```

Comment:
It's partly a matter of taste but I don't like `if (something == false)` -- I prefer `if (!something)`